website/src/components/Survey/TaskControls.tsx

```diff
@@ -1,4 +1,5 @@
 import { Box, Flex, IconButton, Progress, Tooltip, useColorModeValue } from "@chakra-ui/react";
+import { useTranslation } from "react-i18next";
```

Comment:
```suggestion
import { useTranslation } from "next-i18next";
```

Reply:
Thank you. I've fixed all the `"react-i18next" ` that I added in my commit. However, there's still one in `src/pages/account/index.tsx` that was inserted by @AbdBarho and I don't know if it should be changed.

Reply:
> Thank you. I've fixed all the `"react-i18next" ` that I added in my commit. However, there's still one in `src/pages/account/index.tsx` that was inserted by @AbdBarho and I don't know if it should be changed.

Yes it should be update too

Reply:
Changed. Thank you!